docs/connector-development/testing-connectors/README.md

```diff
@@ -53,18 +53,8 @@ Once the integration test workflow launches, it will append a link to the workfl
 
 Integration tests can also be manually requested by clicking "[Run workflow](https://github.com/airbytehq/airbyte/actions?query=workflow%3Aintegration-test)" and specifying the connector and GitHub ref.
 
-### 4. Requesting GitHub PR publishing Docker Images
 
-In order for users to reference the new versions of a connector, it needs to be published and available in the [dockerhub](https://hub.docker.com/r/airbyte/source-sendgrid/tags?page=1&ordering=last_updated) with the latest tag updated.
-
-As seen previously, GitHub workflow can be triggered by comment submission. Publishing docker images to the dockerhub repository can also be submitted likewise:
-
-Note that integration tests can be triggered with a slightly different syntax for arguments. This second set is required to distinguish between `connectors` and `bases` folders. Thus, it is also easier to switch between the `/test` and `/publish` commands:
-
-* `/test connector=connectors/source-sendgrid` - Runs integration tests for a single connector on the latest PR commit.
-* `/publish connector=connectors/source-sendgrid` - Publish the docker image if it doesn't exist for a single connector on the latest PR commit.
-
-### 5. Automatically Run From `master`
+### 4. Automatically Run From `master`
```

Comment:
Two things.
1. Should we not still mention `/test`
2. Should me mention in the merge to master section that the connector will be published?